sensing/pointcloud_preprocessor/docs/distortion-corrector.md

```diff
@@ -2,46 +2,47 @@
 
 ![distortion corrector figure](./image/distortion_corrector.jpg)
 
 ## Inputs / Outputs
 
 ### Input
 
-| Name             | Type                                             | Description      |
-| ---------------- | ------------------------------------------------ | ---------------- |
-| `~/input/points` | `sensor_msgs::msg::PointCloud2`                  | reference points |
-| `~/input/twist`  | `geometry_msgs::msg::TwistWithCovarianceStamped` | twist            |
-| `~/input/imu`    | `sensor_msgs::msg::Imu`                          | imu data         |
+| Name             | Type                                             | Description                        |
+| ---------------- | ------------------------------------------------ | ---------------------------------- |
+| `~/input/points` | `sensor_msgs::msg::PointCloud2`                  | Topic of the distorted pointcloud. |
+| `~/input/twist`  | `geometry_msgs::msg::TwistWithCovarianceStamped` | Topic of the twist information.    |
+| `~/input/imu`    | `sensor_msgs::msg::Imu`                          | Topic of the IMU data.             |
 
 ### Output
 
-| Name              | Type                            | Description     |
-| ----------------- | ------------------------------- | --------------- |
-| `~/output/points` | `sensor_msgs::msg::PointCloud2` | filtered points |
+| Name              | Type                            | Description                         |
+| ----------------- | ------------------------------- | ----------------------------------- |
+| `~/output/points` | `sensor_msgs::msg::PointCloud2` | Topic of the undistorted pointcloud |
 
 ## Parameters
 
 ### Core Parameters
 
-| Name                   | Type   | Default Value | Description                                                 |
-| ---------------------- | ------ | ------------- | ----------------------------------------------------------- |
-| `timestamp_field_name` | string | "time_stamp"  | time stamp field name                                       |
-| `use_imu`              | bool   | true          | use gyroscope for yaw rate if true, else use vehicle status |
+| Name                           | Type     | Default Value | Description                                                 |
+| ------------------------------ | -------- | ------------- | ----------------------------------------------------------- |
+| `timestamp_field_name`         | `string` | `time_stamp`  | Name of time stamp field.                                   |
+| `use_imu`                      | `bool`   | `true`        | Use gyroscope for yaw rate if true, else use vehicle status |
+| `use_3d_distortion_correction` | `bool`   |               | Use 3d correction if true, otherwise use 2d correction      |
 
 ## Assumptions / Known limits
+
+- The node requires that time synchronization works well between the pointcloud, twist, and IMU.
```

Comment:
`well` is a relative term. it is better just to state synchronization as a requirement

Reply:
Thanks, fixed in https://github.com/autowarefoundation/autoware.universe/pull/7137/commits/7d890acdb657e1f2799ab27e0de421d8e1ea8194

Reply:
Grammar...

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/7137/commits/51e96be55b4cd07b28240379f2694500eb1dc3dc